CHANGELOG/CHANGELOG-1.18.md

```diff
@@ -0,0 +1,2 @@
+# v1.18.0 - Changelog since v1.17.12
+To be filled in via a copy of the automatically generated release notes once the v1.18.0 tag is created. (TODO: julianKatz@)
```

Comment:
Do we want to say this, or just redirect to the releases page where the automatically generated notes are going to appear now each time we cut a new version?